src/main/java/seedu/multitasky/storage/StorageDateConverter.java

```diff
@@ -0,0 +1,60 @@
+package seedu.multitasky.storage;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+
+//@@author A0132788U
+/**
+ * Class that implements methods to convert Date to String to enable Storage to store dates in a human-editable form in
+ * the XML file.
+ * Parses String back to Date and returns to Model.
+ */
+public class StorageDateConverter {
+
+    /** Formatter to parse date into a human-editable string to store in the XML file */
+    private DateFormat formatter = new SimpleDateFormat("dd/MM/yy HH:mm");
```

Comment:
The UI display doesn't do mm/dd/yy though, it's displaying as dd mm, yyyy with the month as the word itself and not the number. So I don't think we have a consistency issue from the user point of view if we were to keep it as dd/mm/yy in storage. Also dd/mm/yy is the more universally understood format compared to mm/dd/yy, which only the US uses.